### ISSUE TITLE
Create PR and get approvals for GA4 blog

### ISSUE THREAD

**Issue by cthomasONRR** [DOI-ONRR/blog-nrrd#320]
To be done after the UX consent form blog goes live.

**Comment by eelzi-ONRR** [comment]
Can we fix this when the GA4 PR is made: <img width="485" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNEl5QkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3f3dc07bba29fd4f32e64cff280966d81a74667f/image.png" alt="image.png" />

**Comment by cthomasONRR** [comment]
Still waiting on DOI comms approval. I will create a new issue for final approvals publishing the blog.
